<?xml version="1.0" ?>
<robot name="iqr_ptu2">
  <link name="base_link"/>
  <joint name="ptu2_base_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0 0 0"/>
    <parent link="base_link"/>
    <child link="ptu2_base_link"/>
  </joint>
  <link name="ptu2_base_link">
    <visual>
      <origin rpy="0 0 -1.5707963267948966" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://iqr_ptu2_ros2/urdf/mesh/base.stl"/>
      </geometry>
      <material name="">
        <color rgba="0.2 0.2 0.2 1"/>
      </material>
    </visual>
  </link>
  <link name="ptu2_yaw_link">
    <visual>
      <origin rpy="0 0 -1.5707963267948966" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://iqr_ptu2_ros2/urdf/mesh/yaw.stl"/>
      </geometry>
      <material name="">
        <color rgba="0.2 0.2 0.2 1"/>
      </material>
    </visual>
  </link>
  <joint name="ptu2_yaw_joint" type="revolute">
    <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.023"/>
    <parent link="ptu2_base_link"/>
    <child link="ptu2_yaw_link"/>
    <axis xyz="0 0 1"/>
    <limit effort="0.0" lower="-1.0471975511965976" upper="1.0471975511965976" velocity="0.5235987755982988"/>
  </joint>
  <link name="ptu2_pitch_link">
    <visual>
      <origin rpy="0 0 -1.5707963267948966" xyz="0 0 0"/>
      <geometry>
        <mesh filename="package://iqr_ptu2_ros2/urdf/mesh/pitch.stl"/>
      </geometry>
      <material name="">
        <color rgba="0.2 0.2 0.2 1"/>
      </material>
    </visual>
  </link>
  <joint name="ptu2_pitch_joint" type="revolute">
    <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.062"/>
    <parent link="ptu2_yaw_link"/>
    <child link="ptu2_pitch_link"/>
    <axis xyz="0 1 0"/>
    <limit effort="0.0" lower="-1.0471975511965976" upper="1.0471975511965976" velocity="0.5235987755982988"/>
  </joint>
  <link name="ptu2_surface"/>
  <joint name="ptu2_surface_joint" type="fixed">
    <origin rpy="0 0 0" xyz="0.0 0.0 0.032"/>
    <parent link="ptu2_pitch_link"/>
    <child link="ptu2_surface"/>
  </joint>
</robot>
